### DISCUSSION TITLE
Should I use EF Core in Uno?

### DISCUSSION THREAD

**Discussion by DavidM29** [unoplatform/uno#18130] [Q&A] [answered]
Hello, as Uno Sqlite sample uses sqlite-net-pcl, is that the recommended approach for creating a local database? And is it bad to use EF Core?

**Comment by MartinZikmund** [comment, marked as answer]
You should be able to use SQLite or LiteDB in Uno Platform apps. EF Core with SQLite should probably work ok too, but we haven't validated it against latest releases yet (especially worth verifying iOS and WASM support)
